### DISCUSSION TITLE
No module named 'discord'

### DISCUSSION THREAD

**Discussion by AvalonDawn** [Rapptz/discord.py#5927] [Q&A]
This is what I get when I try to run my main.py file. I scanned my files for syntax errors and it says there are not any. 

C:\Users\Dawn\anaconda3\pythonw.exe C:/Users/Dawn/Desktop/house_bot/main.py
Traceback (most recent call last):
  File "C:/Users/Dawn/Desktop/house_bot/main.py", line 2, in <module>
    import discord
ModuleNotFoundError: No module named 'discord'

Process finished with exit code 1

If anyone can help that would be great! I will leave a comment about what happens when I try installing discord.py.

**Comment by AvalonDawn** [comment]
When I type pip install discord.py into my command prompt I get this,
 "C:\Users\Dawn>pip install discord.py
Requirement already satisfied: discord.py in c:\users\dawn\appdata\local\programs\python\python39\lib\site-packages (0.16.12)
Requirement already satisfied: aiohttp<1.1.0,>=1.0.0 in c:\users\dawn\appdata\local\programs\python\python39\lib\site-packages (from discord.py) (1.0.5)
Requirement already satisfied: websockets<4.0,>=3.1 in c:\users\dawn\appdata\local\programs\python\python39\lib\site-packages (from discord.py) (3.4)
Requirement already satisfied: chardet in c:\users\dawn\appdata\local\programs\python\python39\lib\site-packages (from aiohttp<1.1.0,>=1.0.0->discord.py) (3.0.4)
Requirement already satisfied: multidict>=2.0 in c:\users\dawn\appdata\local\programs\python\python39\lib\site-packages (from aiohttp<1.1.0,>=1.0.0->discord.py) (5.0.0)
Requirement already satisfied: async_timeout in c:\users\dawn\appdata\local\programs\python\python39\lib\site-packages (from aiohttp<1.1.0,>=1.0.0->discord.py) (3.0.1)

**Comment by AbstractUmbra** [comment]
You are using a Python version lower than 3.5.3, which is the minimum supported version of Python that discord.py will install on.
Or you are failing to pass the `-U` flag to `pip install`:
`python -m pip install -U discord.py` - `-U` means Upgrade.

Please investigate the above and try again. :)

**Comment by daggy1234** [comment]
You are using `conda` for dependency management instead of pip.

Please either change your Interpreter to a non-conda one or use `conda install discord` to install the library
